addons/spectator/functions/fn_drawTags.sqf

```diff
@@ -130,62 +115,59 @@ _unitTagSize = 1 * GVAR(unitTagScale);
         _time = time - _time;
         _pos = (getPosATLVisual _unit);
-        if(surfaceIsWater _pos) then {_pos = getPosASLVisual _unit;};
+        _pos = if (surfaceIsWater _pos) then { getPosASLVisual _unit };
         _pos set [2,(_pos select 2)+1];
-        if(_time <= 10 && {_campos distance2d _pos <= 500}) then {
+        if (_time <= 10 && {_campos distance2d _pos <= 500}) then {
             drawIcon3D ["\a3\Ui_F_Curator\Data\CfgMarkers\kia_ca.paa", [1,1,1,1 - (0.1 * _time)],_pos, _unitTagSize/2, _unitTagSize/2, 0,_dName, 2,0.04,"PuristaSemibold" ];
-        }
-        else {
+        } else {
             GVAR(killedUnits) set [_forEachIndex,0];
         };
     };
 } foreach GVAR(killedUnits);
-GVAR(killedUnits) = GVAR(killedUnits)  - [0];
+GVAR(killedUnits) = GVAR(killedUnits) - [0];
 
 
 
 if(GVAR(tracers)) then {
     _deleted = false;
     {
         _x params ["_object","_posArray","_last"];
-        if(!isNull _object) then
-        {
+        if (!isNull _object) then {
             private _missile = _object isKindOf "RocketCore" || _object isKindOf "MissileCore";
             private _grenade = !_missile && {_object isKindOf "Grenade"};
 
             _pos = (getPosATLVisual _object);
-            if(surfaceIsWater _pos) then {_pos = getPosASLVisual _object;};
+            _pos = if (surfaceIsWater _pos) then { getPosASLVisual _object };
 
-            if(_missile) then
-            {
+            if (_missile) then {
               drawIcon3D["\x\tmf\addons\spectator\images\missile.paa",[1,0,0,0.7],_pos,0.5,0.5,0,"",1,0.02,"PuristaSemibold"];
 
               reverse _posArray;
               _prev = getPosVisual _object;
               {
                   drawLine3D [_prev,_x,[1,0,0,0.7]];
                   _prev = _x;
-              } forEach _posArray;
+                  false
+              } count _posArray;
               reverse _posArray;
-              if(time > _last) then {
+              if (time > _last) then {
                   _posArray pushBack _pos;
                   _last = time+1;
               };
             };
-            if(_grenade && {_campos distance2d _object <= 300}) then
-            {
+            if (_grenade && {_campos distance2d _object <= 300}) then {
                 _icon = "\x\tmf\addons\spectator\images\grenade.paa";
-                if(_object isKindOf "SmokeShell") then {_icon = "\x\tmf\addons\spectator\images\smokegrenade.paa"};
+                _icon = if (_object isKindOf "SmokeShell") then { "\x\tmf\addons\spectator\images\smokegrenade.paa" };
                 drawIcon3D[_icon,[1,0,0,0.7],_pos,0.5,0.5,0,"",1,0.02,"PuristaSemibold"];
 
-
                 _prev = getPosVisual _object;
                 if(count _posArray <= 0 || speed _object > 0 ) then {
                   reverse _posArray;
                   {
                       drawLine3D [_prev,_x,[1,0,0,0.7]];
                       _prev = _x;
-                  } forEach _posArray;
```

Comment:
Usage of `count` over `forEach` is not preferred: The minimal performance boost gets negated by decreased readability.